.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 node_js:
 - node
+- '8'
```

Comment:
How about we don't explicitly build for node 8? In most cases, our software will either not work on the latest version of node or the oldest version. Not something in between.